### PR TITLE
Write guide articles about IoT with W5500

### PR DESCRIPTION
**Pull request by nkrkv**
Brings three new articles. Two about W5500 connection and another for HTTP GETs

### CODE REVIEW

**Comment by nkrkv** [comment]
Not rebased/squashed yet, but ready from the content point of view.

File: docs/guide/w5500-advanced/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+[MAC address](https://en.wikipedia.org/wiki/MAC_address) which is (roughly) a
+kind of serial number for network devices. All network devices should have a
+unique MAC address. At the very least, a router uses it to lease the same IP
+address to a device when it shuts down and then powers up again.
+
+You should find a personal MAC for your hardware printed on a sticker in a form
+like “DE:AD:BE:EF:FE:ED”. The W5500 chip does not hard-code its MAC as many
+consumer devices do, so you have to provide it manually.
+
+Place a [`xod/net/mac`](https://xod.io/libs/xod/net/mac/) node and bind link its
```
Comment (evgenykochetkov) [inline docs/guide/w5500-advanced/README.md]: > ... and **bind** link its ...
I think "bind" is not needed

File: docs/guide/w5500-advanced/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+## Connecting to LAN with static IP
+
+As an alternative to connecting via DHCP, you may prefer to connect manually
+with specific network parameters. Although the process is a bit more
+complicated, it makes your device more robust in the case of persistent hardware
+setup.
+
+While connecting manually you have to provide your gateway IP, DNS, network
+mask, and own IP explicitly, but on another hand, you make connection quicker
```
Comment (evgenykochetkov) [inline docs/guide/w5500-advanced/README.md]: on **other** hand?

**Comment by nkrkv** [comment]
@evgenykochetkov I’ve done fixes, please, review

**Pull request merged**